test/unit/foreman/renderer/snapshots/ProvisioningTemplate/PXELinux/Preseed_default_PXELinux_Autoinstall.ubuntu_autoinst4dhcp.snap.txt

```diff
@@ -18,8 +18,8 @@
 #
 DEFAULT linux cloud-init autoinstall
 LABEL linux cloud-init autoinstall
-    KERNEL ubuntu/focal/vmlinuz
-    INITRD ubuntu/focal/initrd
-    APPEND ip=dhcp url=http://archive.ubuntu.com:80/ubuntu autoinstall ds=nocloud-net;s=http://archive.ubuntu.com:80/pub/preseed/autoinstall// root=/dev/ram0 ramdisk_size=1500000 fsck.mode=skip
+    KERNEL boot/ubuntu-mirror-rf32u3HGTMZf-vmlinuz
+    INITRD boot/ubuntu-mirror-rf32u3HGTMZf-initrd
+    APPEND ip=dhcp url=http://archive.ubuntu.com:80/ubuntu autoinstall ds=nocloud-net;s=http://foreman.example.com/userdata/ root=/dev/ram0 ramdisk_size=1500000 fsck.mode=skip
```

Comment:
```suggestion
    APPEND ip=dhcp url=http://archive.ubuntu.com:80/ubuntu.iso autoinstall ds=nocloud-net;s=http://foreman.example.com/userdata/ root=/dev/ram0 ramdisk_size=1500000 fsck.mode=skip
```

I guess this would need adjustment too? ;-)

Reply:
Yep, thanks!